holoviews/core/data/pandas.py

```diff
@@ -291,6 +294,16 @@ def dframe(cls, columns, dimensions):
             return columns.data.copy()
 
 
+    @classmethod
+    def array(cls, dataset, dimensions):
+        if not dimensions:
+            dimensions = dataset.dimensions(label='name')
+        else:
+            dimensions = [dataset.get_dimensions(d).name for d in dimensions]
+        inds = [dataset.data.columns.index(dim.name) for dim in dimensions]
+        return dataset.data.values[:, inds]
+
```

Comment:
I don't quite see how this relates to the python 2 vs 3 sorting issue. Is this method implementation an unrelated fix?

Reply:
Yes, accidentally pushed this. Will revert.